melodies_monet/_cli.py

```diff
@@ -137,7 +137,7 @@ def get_aeronet(
     start_date: str = typer.Option(..., "-s", "--start-date", help="Start date."),
     end_date: str = typer.Option(..., "-e", "--end-date", help="End date."),
     daily: bool = typer.Option(False, help="Whether to retrieve the daily averaged data product."),
-    freq: str = typer.Option(None, "-f", "--freq", help="Frequency to resample to."),
+    freq: str = typer.Option("H", "-f", "--freq", help="Frequency to resample to."),
```

Comment:
with this change, there is no way to get the raw data product, but I think that is ok